### PR TITLE
Updated examples

### PR DESCRIPTION
**Pull request by vkobinski**

### CODE REVIEW

File: examples/tree.py [hunk 1]
```diff
@@ -1,36 +1,21 @@
...
-    value: u24
-
-
-@dataclass
-class Node:
-    left: 'Tree'
-    right: 'Tree'
-
-
-Tree = Node | Leaf
```
Comment (steinerkelvin) [inline examples/tree.py]: keep the expected code that we'll have working in the future commented

File: examples/simple.py [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
 def simple() -> u24:
     x = u24(3)
     y = x - u24(2)
     return y
 
 
-if __name__ == "__main__":
-    translated_simple = bjit(simple)
-    print(simple())
-    print(translated_simple())
```
Comment (steinerkelvin) [inline examples/simple.py]: This is not that relevant, but open an issue to track implementing being able to do this.
I suspect there is something weird going on anyway for this to not work.

File: examples/tree.py (hunk 1 above)
Comment (vkobinski) [reply examples/tree.py]: Ok

Comment (vkobinski) [reply examples/tree.py]: Now the old example works properly.

File: examples/simple.py (hunk 2 above)
Comment (vkobinski) [reply examples/simple.py]: Ok

**Pull request merged**